web/templates/left_sidebar.hbs

```diff
@@ -154,11 +154,11 @@
                     </a>
                     <span class="unread_count"></span>
                 </div>
+                <a class="zoom-out-hide" id="hide-more-direct-messages">
+                    <span> {{t 'back to channels' }}</span>
+                </a>
```

Comment:
After this change, the `direct-messages-section` div has only one child; should we be cleaning that up?

Reply:
Almost certainly. It can probably be wiped and just let the `#direct-messages-section-header` element carry the day.

Reply:
Removed two divs in two different commits, the last commit has the potential to have regressions because of how the selectors were written. I've tried it out for regressions, but having a second set of eyes on it for manual testing would be great. Also mentioned this in the PR description.